flink-runtime/src/main/java/org/apache/flink/runtime/io/network/NetworkEnvironment.java

```diff
@@ -172,6 +173,7 @@ private static void registerNetworkMetrics(MetricGroup metricGroup, NetworkBuffe
 	//  Properties
 	// --------------------------------------------------------------------------------------------
 
+	@VisibleForTesting
 	public ResultPartitionManager getResultPartitionManager() {
```

Comment:
can this be package private?

Reply:
No, it is used by `StreamNetworkBenchmarkEnvironment` from other package atm. It is netty specific test performance benchmark.